docs/schema_script.txt

```diff
@@ -12,7 +12,7 @@ asset.doc.namespace.update :create true :namespace tigerdata :description "Tiger
 asset.doc.type.update :create true :description "Project metadata" :type tigerdata:project :definition < \
  :element -name code -type string -index true -min-occurs 1 -max-occurs 1 -label "The unique identifier for the project" \
  :element -name title -type string -min-occurs 1 -max-occurs 1 -label "A plain-language title for the project" \
- :element -name description -type string -min-occurs 1 -max-occurs 1 -label "A brief description of the project" \
+ :element -name description -type string -min-occurs 0 -max-occurs 1 -label "A brief description of the project" \
```

Comment:
I'm confused. I thought you said it IS required. Doesn't this change it to not being required? "min-occurs 0" means "not required", right?